databallpy/load_data/tracking_data/_to_match_time.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import numpy as np
+
+from databallpy.load_data.metadata import Metadata
+
+def _to_match_time(i:int, metadata:Metadata) -> str:
+    """_summary_
+
+    Args:
+        tracking_data (pd.DataFrame): _description_
+        metadata (Metadata): _description_
+
+    Returns:
+        pd.DataFrame: _description_
+    """
```

Comment:
See comment at `tracab.py`. I dont think you need to go to np.datetime64 here, takes time but we can do all the calculations in seconds. So start periods just 0, 45*60, etc. Than you can loop over the periods and apply the time for a whole period based on specific rules for that period (max/min time etc.)